app/routes/feed[.atom].tsx

```diff
@@ -0,0 +1,43 @@
+import { Feed } from "feed";
+
+import { site } from "~/config";
+
+import eventJson from "../data/events.json";
+
+export function loader() {
+	const feed = new Feed({
+		title: site.title,
+		description: site.description,
+		id: `${site.baseURL}/`,
```

Comment:
The validator was complaining this wasn't in "canonical form" without the trailing `/` 🤷